backend/src/database/databaseConnection.js

```diff
@@ -1,3 +1,4 @@
+const log = require('../facilities/logging.js')(module.filename);
```

Comment:
Are we creating multiple instances of the logger, every time we call this??

Reply:
it is the same logger instance every time.